visual-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
The original example was crafted to show both stylistic rule *and* those that catch subtle mistakes. If possible, can we use config here that does the same?:

```json
{
  "rules": {
    "color-no-invalid-hex": true,
    "declaration-colon-space-after": ["always"],
    "declaration-no-important": true,
    "selector-max-specificity": ["0,1,0", {"severity": "warning"}],
  }
}
```

I've also thrown in a rule that helps control specificity as that was added after we did the original screenshot.

Should we move these files into a `scripts` subfolder?